daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/svalue/Ordering.scala

```diff
@@ -84,10 +85,16 @@ object Ordering extends scala.math.Ordering[SValue] {
         case (STypeRep(xType), STypeRep(yType)) =>
           diff = TypeOrdering.compare(xType, yType)
         case (_: SPAP, _: SPAP) =>
-          throw SError.SErrorCrash("functions are not comparable")
+          throw SError.SErrorCrash(
```

Comment:
I don’t think this is really an internal error. It’s an error in the daml-lf code the user wrote. Fine to keep it for now but I think we should separate this.